static/js/util/util_test.js

```diff
@@ -259,4 +260,27 @@ describe('utility functions', () => {
       ]);
     });
   });
+
+  describe('validationErrorSelector', () => {
+    const invalid = "invalid-input";
+    let errors, keySet;
```

Comment:
Can we move the `let` into each of the tests below, unless there's some data to share between the tests?